lib/cpe_conversion.py

```diff
@@ -10,6 +10,14 @@
 
 import re
 
+def split_cpe_name(cpename: str) -> list[str]:
```

Comment:
I think the requirement for Black formatting complains about a missing line break here.

```suggestion

def split_cpe_name(cpename: str) -> list[str]:
```

(I use a virtualenv with an up-to-date black formatter and run `black .` before any commit; it could be part of pre-commit, too.)